### PR TITLE
MetaStation Engineering lobby update with service desk

### PR DESCRIPTION
**Pull request by nightred**
## About The Pull Request

Engineering was one of the only departments that did not have a service desk. The lobby is connected to a large hallway, Atmos has a service desk right beside them. The lobby is not public so the average person had little way to actually interact with engineering without pushing in to a restricted zone.

engineering lobby revamp 3

The Storage room is also bigger to improve flow when more then one person is getting parts, new tools, or just puttering around.
Security has better visibility in to security, they can see the decontamination room and have a small view in to the SM.
Service Desk for Engineering lobby.

## Why It's Good For The Game
Enhanced presence to the station by engineering.
This lets people interact with engineers in the same way you can with all other departments,  by filling out paperwork.
Engineering security checkpoint now has the ability to monitor who is going in and out of the engine from the lobby.

## Changelog
:cl:
add: Engineering security checkpoint has better view of who is running around.
add: Engineering service desk
tweak: Engineering shared storage has been revamped
/:cl:

### CODE REVIEW

**Comment by trollbreeder** [comment]
Nice. Engineering shared storage looks quite cramped, though, but there's nothing anyone can do about that due to limited space.

**Comment by zxaber** [comment]
Is it possible to actually see the crystal from the sec checkpoint? Are we going to have hallucinating officers be a standard event?

**Comment by nightred** [comment]
The crystal can not be seen from security, they can just see the door to the inner chamber. And the plasma windows block hallucinations I thought. Or at least radiation.

**Comment by kriskog** [comment]
Merge conflict
Im not a fan of splitting up the manufacturing equipment, as you can control the protolathe from the console. If the goal is to make people bump and swap less, it also seems counterproductive.

**Comment by nightred** [comment]
Fixed the conflict, and cleaned up the storage room again.
I feel this flow is better still, all devices on one side, and the doors offset.

**Review by kriskog — CHANGES_REQUESTED** [review]
Stray pipe
image

Dont power the plasmaglass off the engine output, and add cables to the outpost windows.
image

File: html/changelogs/AutoChangeLog-pr-48358.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: "EdgeLordExe"
+delete-after: True
+changes: 
+  - rscadd: "CMC back on meta!"
+  - bugfix: "first aid kits found in medbay clinic actually contain stuff now"
```
[the remark this answers is not in the document]
Comment (nightred) [reply html/changelogs/AutoChangeLog-pr-48358.yml]: I.... What... Well ok, I have no idea what happened here.
I deleted it.

**Comment by nightred** [comment]
Removed the Pipe, and rewired the sec checkpoint.
That would have been bad right of the SM.

**Comment by kriskog** [comment]
Lets give this a go then

**Pull request merged**